src/telemetry/backend-metrics.yaml

```diff
@@ -0,0 +1,218 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# For more information on configuring this file:
+# https://mozilla.github.io/glean/book/reference/yaml/metrics.html
+
+---
+$schema: moz://mozilla.org/schemas/glean/metrics/2-0-0
+
+page:
+  view:
+    type: event
+    description: |
+      A web page view event distinguishable by the page’s URL.
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1896222
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1896222
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - rhelmer@mozilla.com
+    expires: never
+    extra_keys:
+      path:
+        description: The path of the page.
+        type: string
+      user_id:
+        description: Mozilla Accounts user ID.
+        type: string
+      monitor_user_id:
+        description: Mozilla Monitor-specific user ID.
+        type: string
+      session_id:
+        description: An ID that allows us to track “sessions” of the user experience within the product.
+        type: string
+      utm_campaign:
+        description: Attribution campaign.
+        type: string
+      utm_content:
+        description: Attribution content.
+        type: string
+      utm_medium:
+        description: Attribution medium.
+        type: string
+      utm_source:
+        description: Attribution source.
+        type: string
+      utm_term:
+        description: Attribution term.
+        type: string
+      referrer:
+        description: The referring URL, if any.
+        type: string
+      plan_tier:
+        description: Which tier of plan the user is on [Free, Plus]
+        type: string
+
+subscription:
+  activate:
+    type: event
+    description: |
+      A subscription has been activated for this user.
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1896222
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1896222
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - rhelmer@mozilla.com
+    expires: never
+    extra_keys:
+      user_id:
+        description: Mozilla Accounts user ID.
+        type: string
+      session_id:
+        description: An ID that allows us to track “sessions” of the user experience within the product.
+        type: string
+      monitor_user_id:
+        description: Mozilla Monitor-specific user ID.
+        type: string
+      plan_type:
+        description: Which type of plan the user is on ["1 month", "1 year"]
+        type: string
```

Comment:
I thought you were going to start with just `plan_tier` for all metrics (like for the `page.view` metric above) since that's easy to get, and not have `plan_type` initially since that's harder to get (having to talk to the SubPlat API).

Reply:
This was a mistake on my part, thanks. It's not too hard to talk to the subplat API, but I'd rather be consistent for now, we can add `plan_type` later.